src/lib/sw_engine/tvgSwRasterNeon.h

```diff
@@ -53,33 +53,49 @@ static inline bool neonRasterTranslucentRle(SwSurface* surface, const SwRleData*
 {
     auto span = rle->spans;
     uint32_t src;
+    uint32_t align = 0;
+    uint8x8_t *vDst = NULL;
 
     for (uint32_t i = 0; i < rle->size; ++i) {
+        if (!span || !span->len) {
```

Comment:
Please remove here exception. Raster code mostly have many looping, we should avoid instructions (such as exception handling) as possible.  If it does, then we should fix the root cause.